src/core/gltf2importer/gltf2parser_p.h

```diff
@@ -82,7 +82,7 @@ class KUESA_PRIVATE_EXPORT GLTF2Parser
 
     virtual QVector<KeyParserFuncPair> prepareParsers();
     Qt3DCore::QEntity *parse(const QString &filePath);
-    Qt3DCore::QEntity *parse(const QByteArray &jsonData, const QString &basePath);
+    Qt3DCore::QEntity *parse(const QByteArray &jsonData, const QString &basePath, const QString &filename);
```

Comment:
could the file name be optional, since it's only used for export?

Reply:
done